test/any/client/api_matrix-test.es6.js

```diff
@@ -17,91 +17,161 @@
 'use strict';
 
 describe('ApiMatrix', function() {
-  describe('toMatrix()', function() {
-    let apiImporter = org.chromium.apis.web.ApiImporter.create();
-    let apiMatrix = org.chromium.apis.web.ApiMatrix.create({
-      browserAPIs: apiImporter.browserAPIs,
-    });
-    apiImporter.import('Chrome', '55', 'Window', '10', {
-      'Array': ['find'],
-      'Audio': ['stop'],
+  let apiMatrix;
+  beforeEach(function() {
+    let browserApiDao = foam.dao.EasyDAO.create({
+      name: 'browserApiDao',
+      of: org.chromium.apis.web.BrowserAPI,
+      daoType: 'MDAO',
     });
-    apiImporter.import('Edge', '14', 'Window', '10', {
-      'Array': ['find'],
-      'Audio': ['play'],
-    });
-    apiImporter.import('Safari', '10', 'OSX', '601', {
-      'ApplePay': ['about'],
-      'Audio': ['play', 'stop'],
-      'Array': ['find'],
+    browserApiDao.put(org.chromium.apis.web.BrowserAPI.create({
```

Comment:
The "height" of this setup code almost makes me want to undo my request for it ^_^

We should keep the dao create-then-put setup flow, but let's add a helper method outside the `beforeEach()` that constructs these objects. Then all these lines become something like `browserApiDao.put(browserApi('Chrome', '55', 'Windows', '10', 'Array', 'find'))`.